src/main/java/racingcar/controller/RacingGameController.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import racingcar.domain.InvalidCarNameException;
+import racingcar.domain.InvalidGameCountException;
+import racingcar.domain.RacingGame;
+import racingcar.view.RacingGameView;
+
+public class RacingGameController {
+
+    private RacingGame racingGame;
+    private RacingGameView racingGameView;
+    private static final Logger logger = LoggerFactory.getLogger(RacingGameController.class.getName());
+
+    public RacingGameController() {
+        racingGameView = new RacingGameView();
+    }
+
+    public void startRacingGame() {
+        if (isRacingGameSettingIsValid()) {
+            runGames();
+            racingGameView.printResult(racingGame);
+        }
+    }
+
+    private boolean isRacingGameSettingIsValid() {
+        try {
+            String inputCarNames = racingGameView.getCarNames();
+            int getIterationNumber = racingGameView.getIterNo();
+            racingGame = new RacingGame(inputCarNames, getIterationNumber);
+        } catch (InvalidCarNameException | InvalidGameCountException runtimeException) {
+            logger.error(runtimeException.toString());
```

Comment:
exception 의 message 를 로깅하고 싶을때는 getMessage() 를 사용하면 됩니다. 
```suggestion
            logger.error(runtimeException.getMessage());
```

stacktrace 까지 로깅하고 싶다면 아래와 같이 사용하면 됩니다 
```
    logger.error(e.getMessage(), e )
```